source/refined-github.ts

```diff
@@ -16,7 +16,6 @@ import './features/sticky-conversation-list-toolbar.css';
 import './features/always-show-branch-delete-buttons.css';
 import './features/easier-pr-sha-copy.css';
 import './features/repo-stats-spacing.css';
-import './features/emphasize-draft-pr-label.css';
```

Comment:
This looks like a mistake in the merge conflict resolution. This feature was added back in https://github.com/sindresorhus/refined-github/pull/4728

Reply:
I've opened https://github.com/sindresorhus/refined-github/pull/4757 to restore this import